starter-code/index.html

```diff
@@ -1,14 +1,51 @@
 <!DOCTYPE html>
 <html lang="en">
-<head>
-  <meta charset="UTF-8">
-  <meta name="viewport" content="width=device-width, initial-scale=1.0">
-  <meta http-equiv="X-UA-Compatible" content="ie=edge">
-  <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/normalize/5.0.0/normalize.min.css">
-  <link rel="stylesheet" href="./css/style.css">
-  <script type="text/javascript" src="./js/index.js"></script>
-  <title>Ironhack cart</title>
+
+<head <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <meta http-equiv="X-UA-Compatible" content="ie=edge">
+    <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/normalize/5.0.0/normalize.min.css">
+    <link rel="stylesheet" href="https://maxcdn.bootstrapcdn.com/font-awesome/4.7.0/css/font-awesome.min.css">
+    <link rel="stylesheet" href="./css/style.css">
+    <script type="text/javascript" src="./js/index.js"></script>
+    <title>Ironhack cart</title>
 </head>
+
 <body>
+    <div class="container">
+        <h1>Ironhack Merchandising Shop</h1>
+
+        <div id="products">
+            <div class="product">
+                <div class="product-name"><span class="name">IronBubble-head</span></div>
+                <div class="product-price"><span class="price">$25.00</span></div>
+                <div class="product-quantity">
+                    <label for="quantity">QTY</label>
+                    <input class="quantity" type="number" name="quantity" value="" placeholder="0" />
```

Comment:
Little tip, if you don't want to allow negative values, do:
```
<input class="quantity" type="number" name="quantity" value="" placeholder="0" min="0" />
```
Keep in mind this is not a perfect solution.